Resources/Private/Templates/Tea/Index.html

```diff
@@ -8,6 +8,12 @@ <h2>
         </h2>
 
         <table class="table">
+            <thead>
+                <tr>
+                    <th>a</th>
+                    <th>b</th>
```

Comment:
Please let's have meaningful headings here. I propose the localized labels "UID" and "Title".